File: CS/MyRegistrator/MySkinTabHeaderViewInfo.cs
using DevExpress.XtraTab.ViewInfo;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Example_HidePageHeader {
    public class MySkinTabHeaderViewInfo : SkinTabHeaderViewInfo {
        public MySkinTabHeaderViewInfo(BaseTabControlViewInfo viewInfo)
            : base(viewInfo) {
        }
        protected override BaseTabPageViewInfo CreatePage(DevExpress.XtraTab.IXtraTabPage page) {
            return new MyBaseTabPageViewInfo(page);
        }

    }
}

File: VB/MyRegistrator/MySkinTabHeaderViewInfo.vb
Imports Microsoft.VisualBasic
Imports DevExpress.XtraTab.ViewInfo
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace Example_HidePageHeader
	Public Class MySkinTabHeaderViewInfo
		Inherits SkinTabHeaderViewInfo
		Public Sub New(ByVal viewInfo As BaseTabControlViewInfo)
			MyBase.New(viewInfo)
		End Sub
		Protected Overrides Function CreatePage(ByVal page As DevExpress.XtraTab.IXtraTabPage) As BaseTabPageViewInfo
			Return New MyBaseTabPageViewInfo(page)
		End Function

	End Class
End Namespace
